spaceship/lib/spaceship/tunes/app_ratings.rb

```diff
@@ -47,9 +47,94 @@ def unfold_store_fronts(attrs)
         instance_variable_set(:@store_fronts, unfolded_store_fronts)
       end
 
-      # @return (Array) of raw hashes representing user reviews for the given store front (and optional versionId)
+      # @return (Array) of Review Objects
       def reviews(store_front, versionId = '')
-        client.get_reviews(application.apple_id, application.platform, store_front, versionId)
+        raw_reviews = client.get_reviews(application.apple_id, application.platform, store_front, versionId)
+        reviews = []
+        raw_reviews.each do |review|
+          review["value"]["application"] = self.application
+          reviews << AppReview.factory(review["value"])
+        end
+        reviews
+      end
+    end
+
+    class DeveloperResponse < TunesBase
+      attr_reader :id
+      attr_reader :response
+      attr_reader :last_modified
+      attr_reader :hidden
+      attr_reader :state
+      attr_accessor :application
+      attr_accessor :review_id
+
+      attr_mapping({
+        'responseId' => :id,
+        'response' => :response,
+        'lastModified' => :last_modified,
+        'isHidden' => :hidden,
+        'pendingState' => :state
+      })
+
+      def create!(text)
+        client.create_developer_response(app_id: application.apple_id, platform: application.platform, review_id: review_id, response: text)
+      end
+
+      def update(text)
```

Comment:
I think it should also have bang suffix. because, this change is destructive.

`update!`